scripts/deploy-livenet.js

```diff
@@ -0,0 +1,25 @@
+const assert = require("assert");
+
+const { deployBridge } = require("../lib/bridge");
+
+const { NCT_ADDRESS, BRIDGE_CONTROLLER_ADDRESS } = process.env;
+const zeroAddress = "0x0000000000000000000000000000000000000000";
+
+async function deploy() {
+	assert(NCT_ADDRESS, "NCT address needs to be set");
+	const tokenIssuer = BRIDGE_CONTROLLER_ADDRESS || zeroAddress;
```

Comment:
should we also assert bridge controller address?  or is having the zero addr sometimes wanted?